From 095c20ae0c72ea5238bf342f937eed5f4b1acc0d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Yilli=C3=A9?= <yilliee@protonmail.com>
Date: Wed, 6 Apr 2022 14:21:09 +0500
Subject: [PATCH] graphics_utils : Check for ABGR pixel format too

---
 minuitwrp/graphics_utils.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/minuitwrp/graphics_utils.cpp b/minuitwrp/graphics_utils.cpp
index 03b333ac..10b1ba99 100644
--- a/minuitwrp/graphics_utils.cpp
+++ b/minuitwrp/graphics_utils.cpp
@@ -57,6 +57,8 @@ int gr_save_screenshot(const char *dest)
 
 #if defined(RECOVERY_BGRA)
     surface.format = GGL_PIXEL_FORMAT_BGRA_8888;
+#elif defined(RECOVERY_ABGR)
+    surface.format = GGL_PIXEL_FORMAT_ABGR_8888;
 #else
     surface.format = GGL_PIXEL_FORMAT_RGBA_8888;
 #endif
-- 
2.35.1

